plugins/language_perl.lua

```diff
@@ -4,29 +4,26 @@ local syntax = require "core.syntax"
 syntax.add {
   name = "Perl",
   files = { "%.pm$", "%.pl$" },
-  headers = "^#!.*[ /]perl",
+  headers = "^#!.*[ /]perl[%s-w]*",
   comment = "#",
   patterns = {
-    { pattern = "%#.-\n",                 type = "comment"  },
-    { pattern = { '"', '"', '\\' },       type = "string"   },
-    { pattern = { "'", "'", '\\' },       type = "string"   },
-    { pattern = { "qw%(", "%)", '\\' },     type = "string"   },
-    { pattern = { "qw%[", "%]", '\\' },     type = "string"   },
-    { pattern = { "qw%/", "%/", '\\' },     type = "string"   },
-    { pattern = { "qq?%(", "%)",  '\\' },     type = "string"   },
-    { pattern = { "qq?%[", "%]",  '\\' },     type = "string"   },
-    { pattern = { "qq?%/", "%/",  '\\' },     type = "string"   },
-    { pattern = { "^=%w+", "=cut" },   type = "comment" },
+    { pattern = "%#.-\n",                          type = "comment" },
```

Comment:
I know it's completely irrelevant to the PR, but would you mind using `"%#.*"` here?